tool/travis.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Fail fast
+set -e
+
+# Analyze
+dartanalyzer --fatal-warnings lib/*dart lib/messages/*dart test/*dart
```

Comment:
doing everything under lib is potentially redundant. The analyzer will hit all the code that is reachable from the files under lib/*dart which includes the stuff under src. For example running it today we see hints for src/resolver_impl.dart

If I change the glob to something like lib/**/*.dart we end up seeing duplicate hints